tests/e2e/addremoverereplacevmtodefaulttenant_test.go

```diff
@@ -0,0 +1,67 @@
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This test tries to add, remove and replace vm to the _DEFAULT tenant.
+// Expected behavior is that add/rmv/replace vms for _DEFAULT vmgroup should fail.
+
+package e2e
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"github.com/vmware/docker-volume-vsphere/tests/utils/govc"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/misc"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/vmgroup"
+	. "gopkg.in/check.v1"
+)
+
+const (
+	expctdErr = "This feature is not supported for vmgroup _DEFAULT."
+)
+
+type AddRmvReplaceTestSuite struct {
```

Comment:
DefaultTenantTestSuite may be? or anything better than. this name doesn't tell much about the test suite